lib/templates.js

```diff
@@ -7,16 +7,16 @@ const cds = require("@sap/cds");
  * @param {Object} srv The service definition object.
  * @returns {Object} An object containing ORD annotation.
  */
-const fReadORDExtensions = (srv) =>
-  Object.entries(srv)
-    .filter(([key]) => key.startsWith("@ORD.Extensions."))
-    .reduce(
-      (ordExtensions, [key, value]) => ({
-        ...ordExtensions,
-        [key.slice("@ORD.Extensions.".length)]: value,
-      }),
-      {}
-    );
+const readORDExtensions = (srv) =>
```

Comment:
We are calling this function in multiple places in a sequence when generating ord document and every time we have the same input parameter - serviceDefinition. Why not to call this function once when starting generating ORD document and then pass the result as an input parameter of all subsequent functions: _getGroups, _getApiResources, _getEventResources... ?

Reply:
![image](https://github.com/user-attachments/assets/88bf1b81-2f89-4b3e-8b34-5344e7a1fe5d)

Reply:
let's close this PR and fix this as part of the next PR #64